src/pages/fundamentals/eutxo.mdx

```diff
@@ -354,9 +353,9 @@ verification key hash digest or as script hash digest.
 
 Before we move on, let's recap again:
 
-- An _address_ is made of 2 or 3 parts: a _header_, _payment credentials_ and _delegation credentials_;
+- An _address_ is made of either 2 or 3 parts: a _header_, _payment credentials_ and (optionally) _delegation credentials_;
 - The _header_ describes the _type_ of address and the _network_ it is for;
-- The last part, the _delegation credentials_, is optional though <u>highly recommended</u>;
+- Note that the last part, the _delegation credentials_, while optional is still <u>highly recommended</u>;
```

Comment:
Could remove entirely 'The last part' instead of rewording it

Reply:
agree